src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActiveState-test.js

```diff
@@ -133,5 +133,61 @@ describe('elements/content-sidebar/ActiveState/activity-feed/ActiveState', () =>
         );
 
         expect(wrapper.find('[data-testid="comment"]').hasClass('bcs-is-focused')).toEqual(true);
+        expect(wrapper.find('FeedInlineError').length).toBe(0);
+    });
+
+    test('should correctly handle an actively focused task feed item', () => {
+        const wrapper = mount(
+            <ActiveState
+                items={[fileVersion, taskWithAssignment, appActivity]}
+                currentUser={currentUser}
+                activeFeedEntryId={taskWithAssignment.id}
+                activeFeedEntryType={taskWithAssignment.type}
+            />,
+        );
+
+        expect(wrapper.find('[data-testid="task"]').hasClass('bcs-is-focused')).toEqual(true);
+        expect(wrapper.find('FeedInlineError').length).toBe(0);
+    });
+
+    test('should correctly handle an inline error for a comment id being invalid', () => {
+        const wrapper = mount(
```

Comment:
Again, can we introduce a `getWrapper` helper to reduce the per-test boilerplate?